.github/workflows/run-tests.yml

```diff
@@ -153,6 +153,22 @@ jobs:
           threshold_metric: "lines"
           comment_footer: false
 
+      - name: DB Logs
+        id: dblog
+        if: failure()
+        uses: cafuego/command-output@main
+        with:
+          run: |
+            docker compose -f tests/docker-compose.yml exec -T drupal drush watchdog:show
```

Comment:
I think that will return an error if, for example, the site install step failed, because the dblog module would not have been installed yet (line 136).

Reply:
Sure, but then the site install step failed, so we don't care about this one?

Reply:
Will a failure of the DB log step prevent the display of the PHP fatal error in the next step?

Reply:
`fail-fast: true` is not set, so it should happily try to run the next step.